R/duplicates.R

```diff
@@ -1,16 +1,25 @@
-#' Find duplicate rows in a data frame
+#' Keep non-unique rows in a data frame
 #'
-#' Finds duplicate rows within a data frame. Column names can optionally be
-#' provided. If column names aren't provided, the entire data frame (all
-#' columns) will be searched for duplicate rows. If column names are provided,
-#' only the specified columns will be searched for duplicate rows.
+#' Keeps only non-unique rows within a data frame.
 #' 
 #' @param .data A data.frame.
-#' @param ... Optional column names to use when searching for duplicate rows in specific columns.
-#' @return A tibble containing only duplicate rows.
+#' @param ... Optional variables to use when determining non-uniqueness.
+#'  If omitted, will use all variables in the data frame.
+#' @param .keep_all A flag specifying whether to keep all variables in .data.
+#' @return The original data frame with only non-unique rows.
 #' @export
-duplicates <- function(.data, ...) {
+#' @examples 
+# add_column ---------------------------------
```

Comment:
What is this line of code for?

Reply:
Good catch - that `# add_column ------------` line is a paste error!

Reply:
I've just pushed a fix without it